src/utils/errors/application.error.ts

```diff
@@ -10,19 +10,19 @@ export class ApplicationError extends Error {
 
     Error.captureStackTrace(this, this.constructor);
     this.name = this.constructor.name;
-    this.message = message || 'undefined Application Error';
-    this.code = code || 500;
+    this.message = message || 'UNKNOWN';
+    this.code = code || 2;
   }
 }
 
 export class ServerError extends ApplicationError {
   constructor(message?: string, code?: number) {
-    super(message || 'internal Server Error', code || 500);
+    super(message || 'UNKNOWN: server side error', code || 2);
```

Comment:
There’s no need to prefix the message with the status code’s meaning. The grpc protocol already knows the meaning of the code and putting it in the message too is meaningless and digresses from the main point of the message, which is to add detail.

Please remove the status prefix from each message, only the status code should change.